cdl_examples/tpoce_MED-12_ERA5_evaluation_r1i1p1f1_CNRM-MF_CNRM-RCSM6B-SN_v1-r1_mon_198803-198803.cdl

```diff
@@ -0,0 +1,90 @@
+netcdf tpoce_MED-12_ERA5_evaluation_r1i1p1f1_CNRM-MF_CNRM-RCSM6B-SN_v1-r1_mon_198803-198803 {
+dimensions:
+	axis_nbounds = 2 ;
+	x = 567 ;
+	y = 264 ;
+	deptht = 75 ;
+	time = UNLIMITED ; // (1 currently)
+variables:
+	float nav_lat(y, x) ;
+		nav_lat:standard_name = "latitude" ;
+		nav_lat:long_name = "Latitude" ;
+		nav_lat:units = "degrees_north" ;
+	float nav_lon(y, x) ;
```

Comment:
Should be `lat` and `lon` (no nav_)

The crs and native coordinates variables are missing